python/docker-compose.yml

```diff
@@ -1,18 +1,10 @@
-version: '2.1'
+version: "3"
 services:
-  flask:
-    image: quay.io/azavea/flask:1.0-python3.6-slim
-    working_dir: /usr/src
+  database:
+    build:
+      context: .
+      dockerfile: Dockerfile
     ports:
       - "8080:8080"
     volumes:
-      - ./:/usr/src
-    command:
```

Comment:
This goes away because Gunicorn is no longer used to instantiate the application.